README.md

```diff
@@ -41,6 +41,24 @@ The artefact produced produced by the build process can be found under
 
 [`ci-build.sh`](.github/scripts/ci-build.sh) shows the builds tested by the CI.
 
+### Profile-Guided Optimized Build
+
+For the best performance, we recommend using a profile-guided optimized (PGO)
+build. Rust supports [PGO builds](https://doc.rust-lang.org/rustc/profile-guided-optimization.html)
+by hooking into the LLVM profiling infrastructure. The idea is that we gather
+profiling data by running a representative benchmark and then later use the
+profiling data as a feedback on making optimization decisions.
+
+It is recommended to choose the best-performing GC for the profiling step. For
```

Comment:
This is not true according to your results. You can update this section once we have a conclusion.

Reply:
Yeah -- for the time-being it's fine, I think. I can edit it to say something like "We use our best-performing GC for the profiling step in order to get the best performance results", if you want.